packages/theme/styles/_text-editor.scss

```diff
@@ -108,16 +108,20 @@
   object-fit: contain;
 }
 
-.text-editor-highlighted-node-warning {
-  background-color: var(--theme-warning-color);
+.text-editor-highlighted-node-info {
+  background-color: var(--text-editor-highlighted-node-info-background-color);
+  border-bottom: 0.0625rem solid var(--text-editor-highlighted-node-info-border-color);
```

Comment:
what's the purpose of using rem instead of 1px for border?

Reply:
I think it would be better to use rem everywhere for consistency.